### PR TITLE
doc: install/manual-deployment: use sudo on some commands

### PR DESCRIPTION
**Pull request by bstillwell**
Using sudo is necessary on these commands, also a few grammar fixes.

### CODE REVIEW

File: doc/install/manual-deployment.rst [hunk 1]
```diff
@@ -156,28 +156,28 @@ The procedure is as follows:
 
 #. Create a keyring for your cluster and generate a monitor secret key. ::
 
-	ceph-authtool --create-keyring /tmp/ceph.mon.keyring --gen-key -n mon. --cap mon 'allow *'
+	sudo ceph-authtool --create-keyring /tmp/ceph.mon.keyring --gen-key -n mon. --cap mon 'allow *'
```
Comment (tchaikov) [inline doc/install/manual-deployment.rst]: one don't need to be root to create the keyring

File: doc/install/manual-deployment.rst [hunk 2]
```diff
@@ -156,28 +156,28 @@ The procedure is as follows:
...
 
 
 #. Generate an administrator keyring, generate a ``client.admin`` user and add
    the user to the keyring. :: 
 
-	ceph-authtool --create-keyring /etc/ceph/ceph.client.admin.keyring --gen-key -n client.admin --set-uid=0 --cap mon 'allow *' --cap osd 'allow *' --cap mds 'allow'
+	sudo ceph-authtool --create-keyring /etc/ceph/ceph.client.admin.keyring --gen-key -n client.admin --set-uid=0 --cap mon 'allow *' --cap osd 'allow *' --cap mds 'allow'
```
Comment (tchaikov) [inline doc/install/manual-deployment.rst]: ditto.

File: doc/install/manual-deployment.rst [hunk 3]
```diff
@@ -156,28 +156,28 @@ The procedure is as follows:
...
 
 
 #. Add the ``client.admin`` key to the ``ceph.mon.keyring``. :: 
 
-	ceph-authtool /tmp/ceph.mon.keyring --import-keyring /etc/ceph/ceph.client.admin.keyring
+	sudo ceph-authtool /tmp/ceph.mon.keyring --import-keyring /etc/ceph/ceph.client.admin.keyring
```
Comment (tchaikov) [inline doc/install/manual-deployment.rst]: ditto

File: doc/install/manual-deployment.rst [hunk 4]
```diff
@@ -156,28 +156,28 @@ The procedure is as follows:
...
 
 
 #. Generate a monitor map using the hostname(s), host IP address(es) and the FSID. 
    Save it as ``/tmp/monmap``:: 
 
-	monmaptool --create --add {hostname} {ip-address} --fsid {uuid} /tmp/monmap
+	sudo monmaptool --create --add {hostname} {ip-address} --fsid {uuid} /tmp/monmap
```
Comment (tchaikov) [inline doc/install/manual-deployment.rst]: neither does this command needs the root privilege.

File: doc/install/manual-deployment.rst [hunk 5]
```diff
@@ -156,28 +156,28 @@ The procedure is as follows:
...
 
    For example::
 
-	monmaptool --create --add node1 192.168.0.1 --fsid a7f64266-0894-4f1e-a635-d0aeaca0e993 /tmp/monmap
+	sudo monmaptool --create --add node1 192.168.0.1 --fsid a7f64266-0894-4f1e-a635-d0aeaca0e993 /tmp/monmap
```
Comment (tchaikov) [inline doc/install/manual-deployment.rst]: ditto

File: doc/install/manual-deployment.rst [hunk 6]
```diff
@@ -192,11 +192,11 @@ The procedure is as follows:
 
 #. Populate the monitor daemon(s) with the monitor map and keyring. ::
 
-	ceph-mon [--cluster {cluster-name}] --mkfs -i {hostname} --monmap /tmp/monmap --keyring /tmp/ceph.mon.keyring
+	sudo ceph-mon [--cluster {cluster-name}] --mkfs -i {hostname} --monmap /tmp/monmap --keyring /tmp/ceph.mon.keyring
```
Comment (tchaikov) [inline doc/install/manual-deployment.rst]: it could be the "ceph" user to initialize the data directory for thie monitor?

**Comment by tchaikov** [comment]
Bryan, could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#preparing-and-sending-patches

and also add a Signed-off-by line at the end of your commit message? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#signing-contributions .

File: doc/install/manual-deployment.rst (hunk 1 above)
Comment (bstillwell) [inline doc/install/manual-deployment.rst]: Agreed

File: doc/install/manual-deployment.rst (hunk 2 above)
Comment (bstillwell) [inline doc/install/manual-deployment.rst]: This command needs to write to /etc/ceph/ceph.client.admin.keyring, so I believe it needs to be run with sudo.

File: doc/install/manual-deployment.rst (hunk 3 above)
Comment (bstillwell) [inline doc/install/manual-deployment.rst]: Agreed, sudo is not needed here.

File: doc/install/manual-deployment.rst (hunk 4 above)
Comment (bstillwell) [inline doc/install/manual-deployment.rst]: Agreed

File: doc/install/manual-deployment.rst (hunk 5 above)
Comment (bstillwell) [inline doc/install/manual-deployment.rst]: Agreed

File: doc/install/manual-deployment.rst (hunk 6 above)
Comment (bstillwell) [inline doc/install/manual-deployment.rst]: I believe you're right that starting with infernalis this needs to be run as the "ceph" user.

**Comment by bstillwell** [comment]
Thanks for the links for submitting patches.  I'll get this fixed up soon.

**Pull request merged**